### PR TITLE
Audio manager

### PR DESCRIPTION
**Pull request by piskorzm**

### CODE REVIEW

File: Assets/Scripts/Environment/AudioManager.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        Play("MainTheme");
+    }
+
+
+    public void Play(string name)
+    {
+        Sound s = Array.Find(sounds, sound => sound.name == name);
+        if (s == null)
+        {
+            Debug.LogWarning("Sound: " + name + " not found!");
```
Comment (PeterPwlk) [inline Assets/Scripts/Environment/AudioManager.cs:42]: Can be string interpolation :P

File: Assets/Scripts/Environment/AudioManager.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using UnityEngine;
+
+public class AudioManager : MonoBehaviour
+{
+
+    [SerializeField] private Sound[] sounds;
```
Comment (PeterPwlk) [inline Assets/Scripts/Environment/AudioManager.cs:7]: Maybe hashmap would be better? As we are using strings as Id's.

File: Assets/Scripts/Environment/AudioManager.cs (hunk 1 above)
Comment (piskorzm) [reply Assets/Scripts/Environment/AudioManager.cs:46]: NO

File: Assets/Scripts/Environment/AudioManager.cs (hunk 2 above)
Comment (piskorzm) [reply Assets/Scripts/Environment/AudioManager.cs:7]: yaaaaz

**Pull request merged**